frontend/src/cljs/imas_seamap/story_maps/subs.cljs

```diff
@@ -5,3 +5,14 @@
 
 (defn featured-maps [db _]
   (get-in db [:story-maps :featured-maps]))
+
+(defn featured-map [db _]
+  (get-in db [:story-maps :featured-map]))
+
+(defn featured-map-open?
+  "There's a reason that we separately track the drawer open/closed state; if we
+   just told the drawer to be open when the featured map was not nil, then when we
+   erase the featured map the drawer would have errors as it's closing saying it
+   has no title/content to display while in the closing state."
```

Comment:
I love that you've included this, thanks; I suspect it can be a bit clearer though.